t_frang/test_http_trailer_split_allowed.py

```diff
@@ -0,0 +1,165 @@
+"""Tests for Frang directive `http_trailer_split_allowed`."""
+from framework import tester
+from helpers import dmesg
+
+
+
+COUNT_WARNINGS_OK = 1
+
+
+ERROR_MSG = 'Frang limits warning is not shown'
+
+RESPONSE_CONTENT = """HTTP/1.1 200 OK\r
+Content-Length: 0\r\n
+Connection: keep-alive\r\n\r\n
+"""
+
+TEMPESTA_CONF = """
+cache 0;
+listen 80;
+
+
+frang_limits {
+    http_trailer_split_allowed true;
+}
+
+
+server ${server_ip}:8000;
+"""
+
+WARN = 'frang: HTTP field appear in header and trailer'
+
+ACCEPTED_REQUESTS = 'POST / HTTP/1.1\r\n' \
+                'Host: debian\r\n' \
+                'Transfer-Encoding: gzip, chunked\r\n' \
+                '\r\n' \
+                '4\r\n' \
+                'test\r\n' \
+                '0\r\n' \
+                'HdrTest: testVal\r\n' \
+                '\r\n' \
+                'GET / HTTP/1.1\r\n' \
+                'Host: debian\r\n' \
+                'HdrTest: testVal\r\n' \
+                'Transfer-Encoding: chunked\r\n' \
+                '\r\n' \
+                '4\r\n' \
+                'test\r\n' \
+                '0\r\n' \
+                '\r\n' \
+                'POST / HTTP/1.1\r\n' \
+                'Host: debian\r\n' \
+                'HdrTest: testVal\r\n' \
+                '\r\n' \
+
+NOT_ACCEPTED_REQUEST = 'GET / HTTP/1.1\r\n' \
+                'Host: debian\r\n' \
+                'HdrTest: testVal\r\n' \
+                'Transfer-Encoding: chunked\r\n' \
+                '\r\n' \
+                '4\r\n' \
+                'test\r\n' \
+                '0\r\n' \
+                'HdrTest: testVal\r\n' \
+                '\r\n' \
```

Comment:
In my understanding the request should be blocked by default, but _accepted_ with `http_trailer_split_allowed true`, so probably we have a bug in Tempesta FW and the test is broken

Reply:
yes, we thought so too and asked it here...  https://tempesta-tech.slack.com/archives/G6DJW0QNM/p1660049379477609

Reply:
Oops, I missed the message. This is apparently a bug https://github.com/tempesta-tech/tempesta/issues/1725 . Need to fix the test (invert the logic) and mask it by the task

Reply:
`http_trailer_split_allowed true` option logic is fixed in https://github.com/tempesta-tech/tempesta/pull/1750

Reply:
Done: test logic is inverted too

Reply:
Done. I parametrized these tests and test logic is inverted too.